sources/osgGA/FirstPersonManipulator.js

```diff
@@ -56,14 +57,17 @@ FirstPersonManipulator.prototype = MACROUTILS.objectInherit( Manipulator.prototy
         this._zoom = new OrbitManipulator.Interpolator( 1 );
 
         this._stepFactor = 1.0; // meaning radius*stepFactor to move
-        this._target = Vec3.create();
         this._angleVertical = 0.0;
         this._angleHorizontal = 0.0;
 
         // tmp value use for computation
         this._tmpGetTargetDir = Vec3.create();
 
-        this._rotBase = Matrix.create();
+        // vr controls
+        this._vrEnable = false;
```

Comment:
We should not have two differents way to represent rotation inside the Manipulator. It would make more sense to move angleVertical/angleHorizontal in mouse device, same logic for vrRot/Trans in webVR device, and using only simple representation of translation / rotation inside the manipulator. That would be easier for the manipulator, not sure it would work without issue.
Did you think about that or is it too complicated for the current state of the manipulator ?